modules/number-picker/main/index.coffee

```diff
@@ -101,7 +107,7 @@ class NumberPicker extends hx.EventEmitter
     if arguments.length > 0
       @_.min = val
       @selectInput.attr('min', val)
-      checkValue(this, this)
+      @value(@value())
```

Comment:
It's used to `clamp` the value - I guess we could check here if the value is smaller than the min and then do this but I figured as we do the check inside `@value` anyway it made sense to use it.